pkg/network/node/ovscontroller.go

```diff
@@ -673,8 +673,10 @@ func (oc *ovsController) UpdateVXLANMulticastFlows(remoteIPs []string) error {
 // FindPolicyVNIDs returns the set of VNIDs for which there are currently "policy" rules
 // in OVS. (This is used to reinitialize the osdnPolicy after a restart.)
 func (oc *ovsController) FindPolicyVNIDs() sets.Int {
-	_, policyVNIDs := oc.findInUseAndPolicyVNIDs()
-	return policyVNIDs
+	inUseVNIDs, policyVNIDs := oc.findInUseAndPolicyVNIDs()
+	// we need to incldue inUseVNIDs as well because if a namespace has only a deny all
+	// rule, policyVNIDs won't include it.
```

Comment:
can you remove this comment and merge it with the description of the function above (which is not incorrect with your change)

Reply:
Should be fixed now.